lib/constructs/interface.js

```diff
@@ -208,6 +260,23 @@ class Interface {
   }
 
   implements(source) {
+    const iface = this.ctx.interfaces.get(source);
+    if (!iface) {
+      if (this.ctx.options.suppressErrors) {
+        return;
+      }
+      throw new Error(`${source} interface not found (used as a mixin for ${this.name})`);
+    }
+
+    // TODO: are these actually allowed by the spec?
+    if (!this.ctx.options.suppressErrors) {
```

Comment:
Remove this conditional. `suppressErrors` should only be used for stuff we know doesn't work but we know is "safe" to ignore.